src/XRP/model/xrp-payment-channel-create.ts

```diff
@@ -42,13 +49,26 @@ export default class XrpPaymentChannelCreate {
       destinationTag,
       xrplNetwork == XrplNetwork.Test || xrplNetwork == XrplNetwork.Dev,
     )
+    if (!destinationXAddress) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'Cannot construct XAddress from PaymentChannelCreate protobuf `destination` and `destinationTag` fields.',
+      )
+    }
 
     const settleDelay = paymentChannelCreate.getSettleDelay()?.getValue()
+    if (settleDelay === undefined) {
+      throw new XrpError(
+        XrpErrorType.MalformedProtobuf,
+        'PaymentChannelCreate protobuf does not contain `SettleDelay` field.',
```

Comment:
lowercase SettleDelay maybe?  you can see that these fields are all always lower case https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L278